atlasdb-client/src/test/java/com/palantir/atlasdb/table/description/ApiTestSchema.java

```diff
@@ -68,6 +69,20 @@ private static Schema generateSchema() {
             }
         });
 
+        schema.addTableDefinition("AllValueTypesTest", new TableDefinition() {
+            {
+                javaTableName("AllValueTypesTest");
+
+                rowName();
+                rowComponent("component1", ValueType.STRING);
+
+                columns();
+                IntStream.range(0, ValueType.values().length)
+                        .forEachOrdered(typeIndex ->
+                                column("column" + typeIndex, "c" + typeIndex, ValueType.values()[typeIndex]));
```

Comment:
nice! should we do this for the row components as well?

Reply:
Ah, good point - this only tests the named column renderer.

Reply:
Rather annoyingly though I'm going to have to filter out one of string or blob because they have to be at the end. Probably going to write this as a stream of the types that aren't restricted, followed by a single blob component at the end for now.

Reply:
also found a cleaner way to write the above (enums have an `ordinal()` method apparently)